### ISSUE TITLE
Ability to change the git repo path

### ISSUE THREAD

**Issue by OndrejHome** [ansible/galaxy-issues#179]
**Use case example:**
I have repository named 'test' with role named 'testrole' on github.
This role is imported on ansible galaxy as 'testrole' from 'test' repository.
I have decided to rename the github repository from 'test' to 'ansible.test'.
On ansible galaxy I can see possibility to import new 'ansible.test' but I don't 
see the old 'test' repository. When I try to import the role through web interface
it shows running indefinitely and clicking the 'running' I don't see any progress or indication
of any import job. When I change the role name to something else like 'testrole2' 
through ansible galaxy import page the roles will get imported under that alternative name. 
However now I have old 'testrole' which I cannot delete through web interface and new 'testrole2'.

**Workaround:** 
- rename the repository back to old name, 
- then remove the role from ansible galaxy, 
- rename the repository to new desired name and 
- import it into ansible galaxy.

**Expected behaviour:**
- possibility A: Allow deletion of roles that don't have a repository or that the repository is not reachable via web interface.
- possibility B: Allow changing repository path from ansible galaxy

**Comment by s-hertel** [comment]
Hi!

Thanks very much for your submission to Ansible. It sincerely means a lot to us.
This issue was able to be created and has been added to my to-do list.

We may also ask some questions and it may be a while before we can get to this, but we'd like to thank you very much for your time!

We will definitely see your comments on this issue when reading this ticket, but may not be able to reply promptly. You may also wish to join one of our two mailing lists which are very active:

https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code
Thank you once again for this and your interest in Ansible!

**Comment by chouseknecht** [comment]
Duplicate of #187
